modal/exception.py

```diff
@@ -86,6 +86,10 @@ class DeserializationError(Error):
     """Raised to provide more context when an error is encountered during deserialization."""
 
 
+class OutputExpiredError(Error):
```

Comment:
Since this is a specific kind of timeout (and because we previously raised `TimeoutError`), you could make it derive from the more general class maybe.

Reply:
Thanks for pointing that out! just changed.